src/Posts/RepostButton.jsx

```diff
@@ -0,0 +1,170 @@
+const item = props.item;
+
+if (!item) {
+  return "";
+}
+
+const reposts = Social.index("repost", item);
+
+const dataLoading = reposts === null;
+
+const repostsByUsers = Object.fromEntries(
+  (reposts || []).filter((repost) => repost.value.type === "repost").map((repost) => [repost.accountId, repost]),
+);
+
+if (state.hasRepost === true) {
+  repostsByUsers[context.accountId] = {
+    accountId: context.accountId,
+  };
+}
+
+const accountsWithReposts = Object.keys(repostsByUsers);
```

Comment:
`accountsWithReposts ` is unused

Reply:
All of the reposts work is in progress, this commit is to close out the blog post ui changes (issue)[https://github.com/near/near-discovery/issues/993], however the work is inside of the "Reposts" button and the repost work itself is incomplete.